packages/now-build-utils/src/detect-builders.ts

```diff
@@ -309,6 +309,13 @@ export async function detectBuilders(
     options
   );
 
+  if (frontendBuilder && framework === 'redwoodjs') {
+    // RedwoodJS uses the /api directory differently so we must
+    // clear any existing builders and only use `@vercel/redwood`.
+    builders.length = 0;
```

Comment:
TIL

Reply:
Its a bit of a hack but it avoids changing `const builders` to `let builders` 😅